Add files via upload

CloudF1sh opened the pull request:
Currently getting a 401 error and does not link to the database

lildonpancho committed:
Merge pull request #41 from PDXPythonPirates/dev-main

Merge dev-main into main

CloudF1sh committed:
Add files via upload

Currently getting a 401 error and does not link to the database

SalMireles commented:
Can you add a description on the changes that are happening in this PR @CloudF1sh?

SalMireles reviewed (changes requested):
looks like old changes made it in here. There was a mention of this being pushed incorrectly? You could push a new branch if you don't want to wipe old changes that don't belong here.

CloudF1sh committed:
Updated with registration page

The registration page is functional, but still struggling to connect with the database.  Once this is resolved, I will be adding the user's personal page, accessible post login.

SalMireles commented:
Discussed offline:

The test checks for the home page to boot properly route “/” and for some reason it’s failing. The major thing I see now is that you have a lot of old files and code meshed in the PR. This is likely from not basing and updating your local git before pushing. Ideally you should only have changes you made in this PR. Would it be possible to add these to a new PR that is updated with the current dev-main?



The general sequence:
git checkout dev-main
git pull origin dev-main
git checkout -b <name of brach with proper fomatting as scott suggested>
add new code
git add files
git commit -m “<desciptive name of commit>.“. In general commits should be short and sweet. Right now there is one single commit
git push origin <name of branch>
Sometimes it pushes to main so make sure it’s based on dev-main on github. To do this edit the title and change the base from main to dev-main
Update the title to match the code purpose
Update the description of the github page to fully describe all the edits so the reviewer knows exactly what to expect without having to decipher the purpose of the PR by reading the code
